src/App.jsx

```diff
@@ -2,16 +2,19 @@ import { BrowserRouter as Router, Route, Routes } from 'react-router-dom'
 import Home from './pages/Home.jsx'
 import Playground from './pages/Playground.jsx'
 import Registration from './pages/Registration.jsx'
+import { AuthProvider } from './context/AuthContext.jsx'
 
 const App = () => {
   return (
-    <Router>
-      <Routes>
-        <Route path="/" element={<Home />} />
-        <Route path="/playground" element={<Playground />} />
-        <Route path="/register" element={<Registration />} />
-      </Routes>
-    </Router>
+    <AuthProvider>
+      <Router>
+        <Routes>
+          <Route path="/" element={<Home />} />
```

Comment:
Make a `authRoute` component which redirects to registration or just simply toasts a "Please register to access this" infom